index.bs

```diff
@@ -28,61 +28,91 @@ url: http://iso.org/#; spec: ISOBMFF; type: dfn;
 	text: VisualSampleEntry
 
 url: http://iso.org/#; spec: ISOBMFF; type: property;
-	text: colr
-	text: pasp
-	text: stsd
 	text: bitr
+	text: colr
 	text: ctts
-	text: sgpd
 	text: nclx
+	text: pasp
+	text: sgpd
+	text: stsd
 
 url: http://iso.org/#; spec: RFC6381; type: property;
 	text: codecs
 
 url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#; spec: AV1; type: dfn;
```

Comment:
Can we get rid of this PDF link?

Reply:
The HTML version of the spec has been removed from the argon site since I've been away.